src/redux/actions.js

```diff
@@ -19,8 +18,3 @@ export const listenToQueries = () => (
     });
   }
 );
-
-export const deleteVisible = (index) => ({
```

Comment:
ah trimming at 40 makes perfect sense and is an easier implementation. Will cause an edge case when there are more than 40 supposed to be visible though 👍